docs/topics/shell.rst

```diff
@@ -53,6 +53,36 @@ this::
 
 Where the ``<url>`` is the URL you want to scrape.
 
+:command:`shell` also works for local files. This can be handy if you want
+to play around with a local copy of a web page. :command:`shell` understands
+the following syntaxes for local files::
+
+    # UNIX-style
+    scrapy shell ./path/to/file.html
+    scrapy shell ../other/path/to/file.html
+    scrapy shell /absolute/path/to/file.html
+
+    # File URI
+    scrapy shell file:///absolute/path/to/file.html
+
+.. warning:: :command:`shell` will interpret ``index.html`` as a domain name,
+   not as a relative path to a local file, and will trigger a DNS lookup error::
```

Comment:
What about making this a note, not a warning? It'd be also good to explain why does it work the way it works, otherwise we'll get pull requests to 'fix the limitation'.